Removal of ngx-text-diff package

The Renovate pull requests have found an artefact issue with `package-lock.json`:

```
npm ERR! code EINVALIDTAGNAME
npm ERR! Invalid tag name "^>=6.0.0" of package "@angular/cdk@^>=6.0.0": Tags may not have any characters that encodeURIComponent encodes.

npm ERR! A complete log of this run can be found in:
npm ERR!     /tmp/renovate-cache/others/npm/_logs/2022-05-27T12_36_03_963Z-debug-0.log

```

This issue comes from the package `ngx-text-diff` because it has set its peer dependencies using a legacy syntax. This issue explains more: https://github.com/ABenassi87/ngx-text-diff/issues/62

However, this package isn't actually used, so the simplest way to resolve this is to remove it from the `package.json` list completely.

--- a/package.json
+++ b/package.json
@@ -76,7 +76,6 @@
     "ngx-image-cropper": "3.2.1",
     "ngx-json-viewer": "2.4.0",
     "ngx-skeleton-loader": "2.4.4",
-    "ngx-text-diff": "^0.6.0",
     "ngx-toastr": "13.1.0",
     "prismjs": "^1.27.0",
     "rxjs": "^6.6.3",

--- a/src/app/modules/catalogue/catalogue.module.ts
+++ b/src/app/modules/catalogue/catalogue.module.ts
@@ -158,7 +158,6 @@ import { BulkDeleteModalComponent } from '@mdm/modals/bulk-delete-modal/bulk-del
 import { ModelMergingComponent } from '@mdm/model-merging/model-merging.component';
 import { ModelsMergingGraphComponent } from '@mdm/models-merging-graph/models-merging-graph.component';
 import { VersioningGraphModalComponent } from '@mdm/modals/versioning-graph-modal/versioning-graph-modal.component';
-import { NgxTextDiffModule } from 'ngx-text-diff';
 import { ReferenceDataComponent } from '@mdm/referenceData/reference-data.component';
 import { ReferenceDataDetailsComponent } from '@mdm/referenceData/reference-data-details/reference-data-details.component';
 import { ReferenceDataTypeComponent } from '@mdm/shared/reference-data-type/reference-data-type.component';
@@ -381,7 +380,6 @@ import { SkeletonBadgeComponent } from '@mdm/utility/skeleton-badge/skeleton-bad
     HttpClientModule,
     ImageCropperModule,
     ReactiveFormsModule,
-    NgxTextDiffModule,
     SharedModule,
     ToastrModule.forRoot({
       timeOut: 30000,
@@ -550,4 +548,4 @@ import { SkeletonBadgeComponent } from '@mdm/utility/skeleton-badge/skeleton-bad
     SkeletonBadgeComponent
   ]
 })
-export class CatalogueModule { }
+export class CatalogueModule {}

